core/src/main/scala/libretto/ClosedLib.scala

```diff
@@ -43,23 +43,23 @@ class ClosedLib[
         out(f)
     }
 
-  implicit class ClosedLinearFunctionOps[A, B](self: A -⚬ B) {
+  extension [A, B](ƒ: A -⚬ B) {
     def curry[A1, A2](using ev: A =:= (A1 |*| A2)): A1 -⚬ (A2 =⚬ B) =
-      dsl.curry(ev.substituteCo[λ[x => x -⚬ B]](self))
+      dsl.curry(ev.substituteCo[λ[x => x -⚬ B]](ƒ))
 
     def uncurry[B1, B2](using ev: B =:= (B1 =⚬ B2)): (A |*| B1) -⚬ B2 =
-      dsl.uncurry(ev.substituteCo(self))
+      dsl.uncurry(ev.substituteCo(ƒ))
   }
 
-  implicit class FocusedOnFunctionCo[F[_], A, B](f: FocusedCo[F, A =⚬ B]) {
+  extension [F[_], A, B](f: FocusedCo[F, A =⚬ B]) {
```

Comment:
I would add comments to both extensions as it's imho not clear what this is.

But as I don't understand it fully I don't even know what the comments should read…

Reply:
Focusing on the in-port/out-port of a function object `A =⚬ B`.